wgpu-core/Cargo.toml

```diff
@@ -36,6 +36,11 @@ serde = { version = "1.0", features = ["serde_derive"], optional = true }
 smallvec = "1.0"
 vec_map = "0.8"
 
+[dependencies.wgt]
```

Comment:
is `serde` still needed as a `wgpu-core` dependency? Perhaps, we could just make it so `"serde" = ["wgpu-types/serde"]` instead

Reply:
I think `serde` will still be needed in `wgpu-core` because not all types will move into `wgpu-types`

Reply:
It will be problematic to have wgpu-core's serde feature to activate the wgpu-types one... I would be ok if it doesn't, unless it is required to do so. I.e. if the user wants to serialize wgpu-types, they add it as a dependency and enable serde for it.